faithcomesbyhearing/sofria-cli
# FCBHDBP-564 uploader (python) / sofria - text loading issues should not result in failure

## Description
Handle missing verses by printing a warning message while detecting and failing on duplicate verses.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-564

## How Do I QA This
1. You should run the following tests and they have to pass:
```shell
node biblebrain_uploader.js run test/input/Spanish_N2SPNBDA_USX/ --populate-db=./output/SPNBDAO_ET-db/SPNBDAO_ET.db --generate-json=./output/SPNBDAO_ET-json
```
- Outcome

```shell
Generate JSON => Generate JSON filese - Start process..
Populate DB - Start process..
Populate DB => Complete:  Mateo
Populate DB => success
Generate JSON => File list (1) processing completed
Generate JSON => Complete USX File: test/input/Spanish_N2SPNBDA_USX/040MAT.usx
load verses success, rowcount 1068
load tableContents success, rowcount 1
```

The above test should create 28 .json files as well

2. You should run the following tests and they have to pass:
```
node tests/gqlquery_verses_by_usx.test.js
```
